config.js

```diff
@@ -1252,6 +1254,23 @@ var config = {
     // dialInConfCodeUrl: 'https://conference-mapper.jitsi.net/v1/access',
     //
 
+    // Generic IFrame to be used for any application you like. You can name the menu entries in "/lang/main.json"
+    // The link supports template strings for:
+    //  - {room} - The room id
+    //  - {lang} - The ISO Language tag
+    //
+    // sharedIFrameConfig: {
+    //     whiteboard: { // The Id and displayed name of the sharedIframe
```

Comment:
![image](https://user-images.githubusercontent.com/13590797/177948490-69f8247e-1a9d-465b-9d8d-412878793e8e.png)
![image](https://user-images.githubusercontent.com/13590797/177948533-c3c2f2ef-2e9c-475f-9dab-0bf93d2299c8.png)
I updated it for all `fakeParticipants` which includes the current Video and Etherpad implementation. 
in 80d92808623944569320568ba8ca378ddb3b30cd and b165f4d3bd4f8d32980359477bae4c7dbb2ae7c1

Reply:
Thank you for updating the Video part as well! I'm not entirely sure about the messages though. If you could make this configurable too, it would be great, but it's ok if you leave it at that.

Reply:
it configurable in the `main.json` via the `notify.shareStart|shareStopped` keys. 
If you have a suggestion on better wording, please name it. I am happy to edit it. I am not that good at wording it clearly

Reply:
Ah awesome! Haha me neither, we'll see with the UX for that. Thanks!